packages/ag-charts-community/src/chart/tooltip/tooltip.ts

```diff
@@ -321,17 +328,24 @@ export class Tooltip {
         toggleClass('hidden', !visible); // Hide if not visible.
         toggleClass('arrow', !!showArrow); // Add arrow if tooltip is constrained.
 
-        if (newClass !== lastClass) {
+        this.updateWrapping();
+
+        if (addCustomClass ?? true) {
```

Comment:
Nit: This looks a bit odd, maybe just default this in the `updateClass()` parameters?